### PR TITLE
Add support for python 3.10

### PR DESCRIPTION
**Pull request by jLebioda**

### CODE REVIEW

**Comment by vildead** [comment]
Fixes #362

File: setup.py [hunk 1]
```diff
@@ -55,24 +65,21 @@
...
-    author_email='valentin.groues@uni.lu',
+    author="Pinar Alper, Valentin Grouès, Yohan Jarosz, Jacek Lebioda, Kavita Rege, Vilem Ded",
+    author_email='lcsb.sysadmins@uni.lu',
     url='https://github.com/elixir-luxembourg/daisy',
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     package_dir={'elixir_daisy':
                      'elixir_daisy'},
     include_package_data=True,
     install_requires=requirements,
+    dependency_links=["https://github.com/jLebioda/mockldap2.git"],
```
Comment (vildead) [inline setup.py:76]: shouldn't this be the elixir-luxembourg repo?

**Review by vildead — CHANGES_REQUESTED** [review]
There is one warning we may get rid of: 
Screenshot from 2022-07-26 13-30-48

Also, did you try to run `makemigrations`? I get update of id fields

File: setup.py (hunk 1 above)
Comment (jLebioda) [reply setup.py]: this should be actually gone :) Just corrected it, thanks!

**Comment by jLebioda** [comment]
> There is one warning we may get rid of: Screenshot from 2022-07-26 13-30-48
> 
> Also, did you try to run `makemigrations`? I get update of id fields

Added migrations, too

**Review by vildead — APPROVED** [review]
Great job! Works well

**Comment by jLebioda** [comment]
Excellent :)

**Pull request merged**